### DISCUSSION TITLE
saitek heay machinary wheel lg4ff install issue

### DISCUSSION THREAD

**Discussion by becherbrook** [berarma/new-lg4ff#102] [Q&A]
Tried following the instructions for installing lg4ff (assuming thats what I need to get oversteer to recognise the saitek heavy machinary wheel/pedals/sidepanel set), and I get this error when I run sudo dkms install /usr/src/new-lg4ff:

Error! Arguments <module> and <module-version> are not specified.
Usage: add <module>/<module-version> or
       add -m <module>/<module-version> or
       add -m <module> -v <module-version>

**Comment by berarma** [comment]
No, you don't need this. Open an issue in Oversteer with information about your wheel. Check there's no similar request already.
